### PR TITLE
[Shim/Beta] Extend Shim to Support New Distro Features

### PR DESCRIPTION
**Pull request by JacksonWeber**
Extend the shim to support
* Live Metrics
* Browser SDK Loader
* Use distro bunyan

Disables Node 14 support in pipelines for now.
Updated tests to account for changes to OpenTelemetry packages.
Updates OpenTelemetry, distro, and exporter depdendencies.
Updated InstrumentationOptions type to extended that found in the distro and removed LogInstrumentationOptions.

### CODE REVIEW

File: src/shim/shim-config.ts [hunk 1]
```diff
@@ -325,6 +329,18 @@ class Config implements IConfig {
             }
         }
 
+        if (this.enableSendLiveMetrics === false) {
```
Comment (hectorhdzg) [inline src/shim/shim-config.ts:332]: This kind of check would be broken the moment we change default values, we need to improve the logic here

File: src/shim/shim-config.ts [hunk 2]
```diff
@@ -203,8 +204,11 @@ class Config implements IConfig {
             options.logInstrumentationOptions = {
```
Comment (hectorhdzg) [inline src/shim/shim-config.ts:204]: Is a good time to remove logInstrumentationOptions  entirely and only use instrumentationOptions, this what will be supported in distro in the future

File: src/shim/shim-config.ts (hunk 1 above)
Comment (JacksonWeber) [reply src/shim/shim-config.ts]: Updated logic to first check if the value is a boolean.

File: src/shim/shim-config.ts (hunk 2 above)
Comment (JacksonWeber) [reply src/shim/shim-config.ts]: Removed the other `logInstrumentationOptions` and cleaned up the tests.

**Review by hectorhdzg — APPROVED** [review]
LGTM

**Pull request merged**